detection_rules/kbwrap.py

```diff
@@ -230,6 +265,39 @@ def kibana_export_rules(
 
             exceptions.append(exception)
 
+    # Parse action connector results from API return
+    action_connectors = []
+    if export_action_connectors:
+        action_results, _ = parse_action_connector_results_from_api(action_corrector_results)
+
+        # Build TOMLException Objects
+        for connector in action_results:
+            try:
+                connector_id = connector.get("id")
+                rule_list = action_connector_rule_table.get(connector_id)
```

Comment:
nit: Is there any reason why some of the processing/parsing up to this point couldn't be just passed to `TOMLActionConnectorContents.from_kibana_resource`  or something instead of doing it all in here? 

At least that way there's a direct conversion from kibana resource? Like would it break some convention or something, just figured id ask.

Reply:
Yea I think we can do that, the Kibana resource is the dictionary. For the action_connector_rule_table (and exception table) bit, we need the table for the 1 to many mapping so this information is not directly contained in Kibana response and needs to be build through the initial parsing pass.